unit_tests/test_data/test_scylla_yaml_builders/rolling-upgrade.yaml

```diff
@@ -13,6 +13,7 @@ stress_cmd_complex_prepare: cassandra-stress user no-warmup profile=/tmp/complex
 stress_cmd_complex_verify_read: cassandra-stress user no-warmup profile=/tmp/complex_schema.yaml  ops'(read1=1,read3=1)' cl=ONE n=5000000 -mode cql3 native -rate threads=1000 -pop seq=1..5000000
 stress_cmd_complex_verify_more: cassandra-stress user no-warmup profile=/tmp/complex_schema.yaml  ops'(read1=1,read2=1,read3=1,update_static=1,update_ttl=1,update_diff1_ts=1,update_diff2_ts=1,update_same1_ts=1,update_same2_ts=1)' cl=ALL n=5000000 -mode cql3 native -rate threads=200 -pop seq=1..5000000
 stress_cmd_complex_verify_delete: cassandra-stress user no-warmup profile=/tmp/complex_schema.yaml  ops'(delete_row=1)' cl=ALL n=500000 -mode cql3 native -rate threads=200 -pop seq=1..500000
+stress_cmd_large_partitions_10m: scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=2000 -clustering-row-count=5555 -clustering-row-size=uniform:1024..2048 -concurrency=5 -connection-count=5 -consistency-level=quorum -rows-per-request=10 -timeout=30s -validate-data
```

Comment:
same as above.

Reply:
@roydahan , there is currently no suitable parameter for that.
It's true that the approach of this test is not optimal.
It can be refactor in the future to have a single parameter that contains a dictionary of stress commands, similar to the dictionary concept of full_partition_scan parameter.